workon

```diff
@@ -40,16 +40,19 @@ run_project() {
   export CURRENT_PROJECT=$1
 
   PROJECT_BASE_DIR="$PROJECT_LIST_DIR/$CURRENT_PROJECT"
-
-  if [[ -f "$PROJECT_BASE_DIR/$CONFIG_FILE_NAME" ]]; then
+  
+  if [[ -d "$PROJECT_BASE_DIR" ]]; then
     cd $PROJECT_BASE_DIR
     PROJECT_ROOT_DIR=$PROJECT_BASE_DIR
-    source $CONFIG_FILE_NAME
+
+    if [[ -f "$CONFIG_FILE_NAME" ]]; then
+      source $CONFIG_FILE_NAME
+    else
+      echo "No configuration file available. Create $PROJECT_BASE_DIR/$CONFIG_FILE_NAME if configuration is needed."
```

Comment:
What do you think about saying that the user should use `$0 add <project> <project_path>` to add the configuration instead? That way, they don't have to think through what could go in there, etc.